ark-testing/src/aspd.rs

```diff
@@ -85,6 +85,24 @@ impl AspD {
 			None => { panic!("Server not yet started!"); }
 		}
 	}
+
+	pub fn run_aspd_command(&self, args: &[&str]) -> anyhow::Result<String> {
+		let mut cmd = self.arkd_cmd.get_cmd();
+		cmd.args(args);
+
+		let result = cmd.output().with_context(|| format!("Failed to execute {:?}", cmd))?;
+
+		// TODO: ensure you write the full command, stdout, stderr to the directory
```

Comment:
Btw, you can have your todos be tagged automatically in Vim by adding this vim config:

```
iabbrev todo TODO(stevenroose)
```
or using lua I guess something like
```
vim.cmd([[
iabbrev todo TODO(stevenroose)
]])
```